cityofaustin/atd-moped
# Update project funding source amount to input only integers

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/8718

This PR adds a MUI TextField wrapper component that limits the inputs to the funding source amounts in the project Funding  tab to positive integers only.

## Testing
**URL to test:** 
https://deploy-preview-647--atd-moped-main.netlify.app/moped/projects/

**Steps to test:**
Go to the Funding tab of any project, click "+ ADD FUNDING SOURCE", and then choose a source from the dropdown.

Try to enter `100.99` for the amount in the new row added. You should not be able to enter the decimal point.

Try to enter +, -, e characters into the field. You should not be able to enter these characters.

Try to enter 1000 or another number without decimals and then save. You should see the row update.

---
#### Ship list
